### ISSUE TITLE
siprtp print_avg_stat() can print NaN/Inf percent loss when packet counters are zero

### ISSUE THREAD

**Issue by tsx543** [pjsip/pjproject#5110]
### Describe the bug

In `pjsip-apps/src/samples/siprtp.c`, `print_avg_stat()` only checks that at least one call is confirmed/active. It does not ensure RX/TX packet counters are non-zero before computing loss percentages:
```c
min_stat.rx.loss*100.0/(min_stat.rx.pkt+min_stat.rx.loss)
...
min_stat.tx.loss*100.0/(min_stat.tx.pkt)
```
For a newly confirmed call with no RTP yet, denominators can be zero, so the printed "percent loss" becomes NaN (0.0/0.0) or Inf (non-zero/0).
This is a sample-app statistics correctness bug, not a memory-safety issue.

### Steps to reproduce

1. Build pjproject samples (includes `siprtp`).
2. Run two `siprtp` instances and establish a call.
3. Print average stats while the call is confirmed but RX/TX packet counters are still zero (or otherwise force zero pkt/loss counters).
4. Observe NaN/Inf in the "percent loss" fields.
Equivalent formula check (same expressions as in `print_avg_stat`):
rx: loss*100.0/(pkt+loss) with pkt=0, loss=0 -> NaN
tx: loss*100.0/pkt with pkt=0, loss=0 -> NaN
tx: loss*100.0/pkt with pkt=0, loss>0 -> Inf

### PJSIP version

2.17-dev

### Context

- Platform: macOS aarch64, clang 17
- Component: sample app `pjsip-apps/src/samples/siprtp.c`, function `print_avg_stat`
- Trigger: confirmed call with zero RTP/RTCP packet counters

### Log, call stack, etc

```shell
rx_percent_loss = NaN   (0/0)
tx_percent_loss = NaN   (0/0)
tx_percent_loss = Inf   (loss>0, pkt==0)
```
